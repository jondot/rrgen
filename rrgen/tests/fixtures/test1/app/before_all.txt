// Both struct should be injected

pub struct World {
}

pub struct World2 {
}
