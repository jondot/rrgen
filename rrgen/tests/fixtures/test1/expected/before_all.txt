// Both struct should be injected

//fields
pub struct World {
}

//fields
pub struct World2 {
}